docs/_data/prague-2021-schedule.yaml

```diff
@@ -0,0 +1,79 @@
+writing_day:
+- time: '10:00'
+  title: 'Platform opens'
+- time: '11:00'
+  title: Morning introduction - Introduction to Sprints & Project Introductions
+- time: '11:15'
+  title: Team up and start working
+- time: '14:30'
+  title: Lunch Break
+- time: '19:00'
```

Comment:
I think that's a really late hour to finish, maybe we can shorten the day a bit to end at 17:00?

Also, @ddbeck @bethaitman when do you want to do the WW tour session?